src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,63 @@
+package duke;
+
+/**
+ * Deals with making sense of the user command
+ *
+ * @author sibinhho99-nus
+ */
+public class Parser {
+    /**
+     * Returns index to delete from delete command
```

Comment:
```suggestion
     * Returns index to delete from delete command.
     * @param Delete command.
     * @return Index of the delete command.
```
Shouldn't JavaDoc method summaries include parameters and return statements?